config/UNet.yaml

```diff
@@ -4,13 +4,13 @@ base: &base
   weight_init: {conv_init: 'normal', conv_scale: 0.02, conv_bias: 0.}
   lambda_rho: 1E-2 # weight for additional rho loss term
   full_scale: True # whether or not to use all 6 of the scales in U-Net
-  batch_size: 1
+  batch_size: 64 # number of samples per training batch (global batch size for distributed training)
```

Comment:
the parentheses statement makes it sound like this is global batch size. I'd just say that this is local batch size, and optionally add that global batch size is (local bs * num ranks).

Reply:
That actually is global batch size. In `train.py` we set local batch size based on num ranks:
https://github.com/NERSC/sc21-dl-tutorial/blob/d2f4a8b4c551db915ca96a064e993217e9faea91/train.py#L224

Reply:
Ah, ok, then this config can be used for strong scaling, but for weak-scaling we need a new config for each scale point, right?

Reply:
Right, so sorta following last year's tutorial setup -- different config for each (global) batchsize

Reply:
ok, thanks.